SIZES.md

```diff
@@ -5,286 +5,286 @@
+| tapT | 42 B | n/a B | n/a B |
+| test | 187 B | 3235 B | -3048 B |
+| throttle | 67 B | n/a B | n/a B |
+| times | 221 B | 326 B | -105 B |
+| toLower | 41 B | 3346 B | -3305 B |
+| toPairs | 60 B | 189 B | -129 B |
+| toPairsIn | 54 B | 166 B | -112 B |
 | toString | 223 B | 3086 B | -2863 B |
 | toUpper | 41 B | 3347 B | -3306 B |
-| transduce | 209 B | 1082 B | -873 B |
-| transduceRight | 140 B | n/a B | n/a B |
-| transpose | 116 B | 204 B | -88 B |
-| trim | 34 B | 317 B | -283 B |
-| tryCatch | 201 B | 487 B | -286 B |
-| type | 109 B | 186 B | -77 B |
-| unapply | 80 B | 161 B | -81 B |
+| transduce | 211 B | 1082 B | -871 B |
+| transduceRight | 139 B | n/a B | n/a B |
+| transpose | 119 B | 204 B | -85 B |
+| trim | 33 B | 317 B | -284 B |
+| tryCatch | 200 B | 487 B | -287 B |
+| type | 111 B | 186 B | -75 B |
+| unapply | 77 B | 161 B | -84 B |
 | unary | 34 B | 430 B | -396 B |
 | uncurryN | 211 B | 618 B | -407 B |
 | unfold | 216 B | 266 B | -50 B |
-| union | 429 B | 2988 B | -2559 B |
-| unionWith | 231 B | 504 B | -273 B |
-| uniq | 571 B | 1908 B | -1337 B |
-| uniqBy | 723 B | 1888 B | -1165 B |
-| uniqWith | 239 B | 320 B | -81 B |
-| unless | 191 B | 338 B | -147 B |
+| union | 433 B | 2988 B | -2555 B |
+| unionWith | 233 B | 504 B | -271 B |
+| uniq | 577 B | 1908 B | -1331 B |
+| uniqBy | 727 B | 1888 B | -1161 B |
+| uniqWith | 240 B | 320 B | -80 B |
+| unless | 190 B | 338 B | -148 B |
 | unnest | 56 B | 1996 B | -1940 B |
-| unnestN | 311 B | n/a B | n/a B |
-| until | 199 B | 351 B | -152 B |
-| unzip | 287 B | n/a B | n/a B |
+| unnestN | 307 B | n/a B | n/a B |
+| until | 200 B | 351 B | -151 B |
+| unzip | 282 B | n/a B | n/a B |
 | unzipWith | 269 B | n/a B | n/a B |
-| update | 217 B | 497 B | -280 B |
-| updateBy | 267 B | n/a B | n/a B |
-| useWith | 199 B | 618 B | -419 B |
-| values | 44 B | 509 B | -465 B |
-| valuesIn | 47 B | 163 B | -116 B |
-| view | 189 B | 259 B | -70 B |
-| when | 191 B | 338 B | -147 B |
-| where | 214 B | 273 B | -59 B |
-| whereEq | 490 B | 2551 B | -2061 B |
-| without | 200 B | 2713 B | -2513 B |
+| update | 220 B | 497 B | -277 B |
+| updateBy | 271 B | n/a B | n/a B |
+| useWith | 197 B | 618 B | -421 B |
+| values | 45 B | 509 B | -464 B |
+| valuesIn | 51 B | 163 B | -112 B |
+| view | 188 B | 259 B | -71 B |
+| when | 190 B | 338 B | -148 B |
+| where | 213 B | 273 B | -60 B |
+| whereEq | 489 B | 2551 B | -2062 B |
+| without | 199 B | 2713 B | -2514 B |
 | xprod | 243 B | 284 B | -41 B |
-| za | 51 B | n/a B | n/a B |
-| zip | 255 B | 273 B | -18 B |
-| zipObj | 50 B | 271 B | -221 B |
-| zipObjDeep | 259 B | n/a B | n/a B |
-| zipWith | 248 B | 383 B | -135 B |
+| za | 49 B | n/a B | n/a B |
+| zip | 256 B | 273 B | -17 B |
+| zipObj | 51 B | 271 B | -220 B |
+| zipObjDeep | 263 B | n/a B | n/a B |
+| zipWith | 249 B | 383 B | -134 B |
 | --- | --- | --- | --- |
 | minimum | 27 B | 42 B | -15 B |
-| median | 197 B | 467 B | -270 B |
+|  | 196 B | 467 B | -271 B |
```

Comment:
after removing `median` function name, this isn't working anymore 😅

```js
const median = require('../../cjs/median')

median.name // ''
```

<img width="129" alt="Screenshot 2020-04-28 at 22 22 44" src="https://user-images.githubusercontent.com/2991847/80528556-ccadc600-899e-11ea-8472-b74b2b693474.png">

please check this line:

https://github.com/nanoutils/nanoutils/blob/d6a6632bef05d5989ae43c7435a5828e2a0fd8e7/misc/cli/size.js#L52

I suggest reverting the changes and leave the comment with description why we shouldn't change this line

Reply:
I wonder if for this particular function we could do this:
```javascript
const median = () => {...};
export default median;
```
Modern browsers understand that syntax and correctly fill `name` field. ES5 version looks a bit suboptimal, but still quite good.

```
var median = function median() {...};
module.exports = median;
``` 

So it's a trade-off. But it doesn't really matter, it's just a detail. I think both solutions would be equally :) What do you think?

Reply:
> I wonder if for this particular function we could do this:
> 
> ```js
> const median = () => {...};
> export default median;
> ```
> 
> Modern browsers understand that syntax and correctly fill `name` field. ES5 version looks a bit suboptimal, but still quite good.
> 
> ```
> var median = function median() {...};
> module.exports = median;
> ```
> 
> So it's a trade-off. But it doesn't really matter, it's just a detail. I think both solutions would be equally :) What do you think?

We import `lib` version of the function so unfortunately this is working this way (if we rely on `cjs`, we won't use it without building it before)

Your solution is working well so this is fine for me 👌

Reply:
Looks like work is done here as well ✅